### PR TITLE
Added links

### PR DESCRIPTION
**Pull request by samyakbvs**
Youtube, Github, Flickr

### CODE REVIEW

**Comment by samyakbvs** [comment]
Issue #240

File: index.html [hunk 1]
```diff
@@ -457,6 +457,27 @@ <h1 class="thin mb0">FOSSASIA <br>Based in Asia.</h1>
 					<i class="ti-vimeo-alt"></i>
 				</a>
 			</li>
+                        <li>
+                                <a href="https://www.instagram.com/fossasia">
+                                        <i class=" ti-instagram" aria-hidden="true"></i>
```
Comment (vaibhavsingh97) [inline index.html]: inconsistent space between `" "`

File: index.html [hunk 2]
```diff
@@ -457,6 +457,27 @@ <h1 class="thin mb0">FOSSASIA <br>Based in Asia.</h1>
...
+                                </a>
+                        </li>
+                        <li>
+                                <a href="https://www.youtube.com/channel/UCQprMsG-raCIMlBudm20iLQ">
+                                        <i class=" ti-youtube  " aria-hidden="true"></i>
```
Comment (vaibhavsingh97) [inline index.html]: same

File: index.html (hunk 1 above)
Comment (vaibhavsingh97) [reply index.html]: use `class="ti-instagram"`

File: index.html [hunk 3]
```diff
@@ -529,6 +550,27 @@ <h3 class="mb40 mb-xs-24">FOSSASIA Google Code-In 2016/17</h3>
 							<i class="ti-vimeo-alt"></i>
 						</a>
 					</li>
+					<li>
+                                                <a href="https://www.instagram.com/fossasia">
+                                                        <i class=" ti-instagram" aria-hidden="true"></i>
```
Comment (vaibhavsingh97) [inline index.html]: same

**Review by vaibhavsingh97 — CHANGES_REQUESTED** [review]
Please fix the suggested changes

**Review by niranjannitesh — APPROVED** [review]
LGTM

**Review by vaibhavsingh97 — APPROVED** [review]
LGTM! 🚀

**Comment by samyakbvs** [comment]
Now is it ready to merge?

**Review by anshumanv — APPROVED** [review]
LGTM

**Comment by saarthakchats** [comment]
There are conflicts

**Pull request merged**